packages/backend/src/api/controllers/liveness/LivenessController.ts

```diff
@@ -39,6 +40,34 @@ export class LivenessController {
         const withAnomalies = calculateAnomaliesPerProject(intervals)
         console.timeEnd(`withAnomalies ${project.projectId.toString()}`)
 
+        if (withAnomalies) {
+          const l2project = layer2s.find((l) => l.id === project.projectId)
+          const anomalies = withAnomalies.anomalies
+          if (l2project?.config.liveness?.duplicatedData?.batchSubmissions) {
+            withAnomalies.batchSubmissions = { ...withAnomalies.stateUpdates }
+            if (anomalies) {
```

Comment:
I am not sure what we should do with the anomalies in this case, maybe let's skip them for now, and keep only the "source" ones